docs/articles/new-expensify/getting-started/Introduction to Expensify Support

```diff
@@ -0,0 +1,175 @@
+Expensify offers comprehensive support for customers of all sizes and accommodates various customer needs, from entrepreneurs to multinational corporations.
+
+**Concierge Support**
+
+- Concierge is available 24/7 and is the first line of support.
+- Users can access Concierge through the Expensify app or via emailconcierge@expensify.com.
```

Comment:
Right?
```suggestion
- Users can access Concierge through the Expensify app or via emailing to [concierge@expensify.com](mailto:concierge@expensify.com).
```

Reply:
Small change:

- Users can access Concierge through the Expensify app or via email by writing to [concierge@expensify.com](mailto:concierge@expensify.com).

Reply:
Updated suggestion